src/schemas/movies.py

```
@@ -3,158 +1,229 @@
+    director_detail_schema_example,
+    certification_schema_example,
+    movie_schema_example,
     movie_detail_schema_example,
-    movie_update_schema_example,
+    movie_create_schema_example,
+    movie_update_schema_example
 )
 
 
-class LanguageSchema(BaseModel):
+class GenreSchema(BaseModel):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)